content/docs/command-reference/metrics/index.md

```diff
@@ -103,8 +103,8 @@ to compare and pick the best performing experiment.
 ## Examples
 
 > This example is based on our
-> [Get Started](/doc/start/experiments#collecting-metrics), where you can find
-> the actual source code.
+> [Get Started](/doc/start/metrics-parameters-plots#collecting-metrics), where
+> you can find the actual source code.
```

Comment:
Good, thanks! But I see another 6 links to the `/doc/start/experiments` in master (you may have gotten a few of those but please check in all of content/ (there's a couple blogs with this). Thanks

Reply:
This makes me wonder if we should change the /doc/start/experiments URL completely, since it has totally new content. Links from 3rd party blogs will now end up in a very different page... And the search ranking may get disrupted (temporarily). Let me check search console... ⌛

Reply:
OK so on the one hand, it has decent traffic. Replacing the URL may disturb that (prob temporarily):

![image](https://user-images.githubusercontent.com/1477535/108916402-d1a9c900-75f3-11eb-9b0b-ba1552cf51f9.png)

On the other, most of the searches are for experiment-related features 😆 so maybe it's better to replace this with that users are already searching, finding, and clicking or wanting to click on:

![image](https://user-images.githubusercontent.com/1477535/108916527-01f16780-75f4-11eb-923f-80d0a030c961.png)

WDYT guys? BTW this is my only major concern (both to replace all links and whether to change the URL, which would imply setting up a redirect) cc @shcheklein

Reply:
Yeah, I had skipped the blog content. I can update those to point to `metrics-parameters-plots`.

I lean towards wanting links to experiments to point to the new page to encourage using the updated experiments features. I did a quick Google search for "dvc experiments" and looked through the first two pages of results as a quick check. I only found one that actually linked directly to the old experiments page, and that one use the old link (see https://towardsdatascience.com/enable-ml-experiments-4ba8c3c8bdc2, which links to https://dvc.org/doc/get-started/compare-experiments). Maybe that old link should redirect to metrics-parameters-plots? But even then, it's probably better to redirect to the current experiments functionality.

Reply:
There's a reference in https://dvc.org/blog/december-20-dvc-heartbeat to the old video. That was removed from the page since it is more focused on experiments. Thoughts on how to handle this reference, but also what to do with that video?

Reply:
> lean towards wanting links to experiments to point to the new page to encourage using the updated experiments features

Sure, depends on the context of each link. Up to you to leave some as-is.
Good to know there's not many 3rd party links so no worries there then.

> /doc/get-started/compare-experiments). Maybe that old link should redirect to metrics-parameters-plots

Everything /doc/get-started* already redirects to /doc/start 👍